test/test.py

```diff
@@ -171,5 +300,21 @@ def test_5_get_info(self):
         self.assertEqual(si.rate, rate)
         self.assertEqual(ei.bits_per_sample, precision)
 
+        torchaudio.set_audio_backend(self.default_audio_backend)
+
+    def _test_5_get_info_soundfile(self):
+        torchaudio.set_audio_backend("soundfile")
+
+        input_path = os.path.join(self.test_dirpath, 'assets', 'sinewave.wav')
+        channels, samples, rate, precision = (1, 64000, 16000, 16)
+        si = torchaudio.info(input_path)
+        self.assertEqual(si.channels, channels)
+        self.assertEqual(si.frames, samples)
+        self.assertEqual(si.samplerate, rate)
+        si_precision = _extract_digits(si.subtype)
```

Comment:
If this is necessary we should at least write out as a todo to make this consistent. That can be done via a wrapper class that uses getattribute etc to align these attributes consistently.

Reply:
It should make sense to standardize on sox since otherwise we'll introduce BC-breaking changes to support this new backend.